README.md

```diff
@@ -2,9 +2,17 @@
 
 [Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This repo is the primary source for core development on the Mattermost platform; it's written in Go and React and runs as a single Linux binary with MySQL or PostgreSQL. A new compiled version is released under an MIT license every month on the 16th.
 
+[Use it for free in Mattermost Cloud](https://mattermost.com/sign-up/) or [deploy on-premises](https://mattermost.com/deploy/).
+
 <img width="1006" alt="mattermost-hero" src="https://user-images.githubusercontent.com/7205829/136107976-7a894c9e-290a-490d-8501-e5fdbfc3785a.png">
 
-Useful resources:
+Learn more about the following use cases with Mattermost:
+
+- [DevSecOps](https://mattermost.com/solutions/use-cases/devops/?source=github-mattermost-server-readme)
```

Comment:
Should we use `utm_source=...` for the query strings?

Reply:
Yes, sorry, changed to use `utm_source` instead of `source`